microsoft-edge/extensions-chromium/publish/api/addons-publish-api-reference.md

```diff
@@ -0,0 +1,371 @@
+title: Add-ons Publish API Reference
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, publish api, add-ons publish api
+---
+# Add-ons Publish API Reference
+
+
+<!-- ====================================================================== -->
+## Get the list of products
+
+Gets a list of all products that belong to the account.  
+
+Overview: [Getting the list of products](using-addons-publish-api.md#getting-the-list-of-products)
+
+### Request
+
+| Method | Request URI |
+|---|---|
+| `GET` | `/products` |
+
+### URI parameters
+
+None.
+
+#### Request headers
+
+* Required.  `Authorization: <auth token>`
+
+#### Request body
+
+None.
+
+### Response
+
+The response includes a list of products, with details for each product.  The template for this response is as follows.
+
+```json
+[
+    {
+        "id": "<productID1>",
+        "name": "<Name of Product 1>",
+        "status": "In review",
+        "lastUpdatedDate": "7/19/2021"
+    },
+    {
+        "id": "<productID2>",
+        "name": "<Name of Product 2>",
+        "status": "In the store",
+        "lastUpdatedDate": "5/21/2021"
+    }
+]
+```
+
+For the first version of a product, the status field has the following values:
+*  In Draft
+*  In Review
+*  In the Store, or Failed
+
+For subsequent submissions, you'll receive either of the following values:
+*  In Review
+*  In the Store, or Failed
+
+### Status codes
+
+This API has the following expected status codes.
+
+| HTTP status code | Description |
+|---|---|
+| 200 | The request is ok |
+| 4XX | For more details see [Error codes](#error-codes). |
+| 5XX | For more details see [Error codes](#error-codes). |
+
+
+<!-- ====================================================================== -->
+## Upload a package to update an existing submission
+
+Uploads a package to update an existing draft submission of an add-on product.
+
+Overview: [Uploading a package to update an existing submission](using-addons-publish-api.md#uploading-a-package-to-update-an-existing-submission)
+
+### Request
+
+| Method | Request URI |
+|---|---|
+| `PUT` | `/products/{productID}/submissions/draft/package` |
+
+#### URI parameters
+
+You can specify the following additional parameters on the request URI:
+
+| URI parameter | Description |
+|---|---|
+| `productID` | Required.  The product ID of the product to which the package must be uploaded. |
+
+#### Request headers
+
+*  Required.  `Authorization: <auth token>`
+
+*  Required.  `Content-Type: application/zip`
+
+#### Request body
+
+* `<Zip package>`
+
+### Response
+
+#### Response headers
+
+*  Location: `/products/\{productID}/submissions/draft/package/operations/\{operationID}`
```

Comment:
Dev: Please in both .md articles globally find "productID" and "operationID".  Check for presence or absence of { braces, $, capitalization.  A draft has "operationsID" with an 's', or variants.  Do the .md files have the right consistency?